noticeboardapp/templates/noticeboard/add_notice.html

```diff
@@ -0,0 +1,39 @@
+{% extends 'homepageapp/base_2.html' %}
+{% load static %}
+{% load tz %}
+
+{% block content %}
+
+<style>
```

Comment:
style is to be used in HEAD section of HTML document. 
Please run the html output through https://validator.w3.org/#validate_by_input